redbot/cogs/audio/converters.py

```diff
@@ -25,50 +27,42 @@
     "get_playlist_converter",
 ]
 
-if TYPE_CHECKING:
-    _bot: Red
-    _config: Config
-else:
-    _bot = None
-    _config = None
-
-_SCOPE_HELP = """
+_SCOPE_HELP: Final[
+    str
+] = """
 Scope must be a valid version of one of the following:
 ​ ​ ​ ​ Global
 ​ ​ ​ ​ Guild
 ​ ​ ​ ​ User
 """
-_USER_HELP = """
+_USER_HELP: Final[
+    str
+] = """
 Author must be a valid version of one of the following:
 ​ ​ ​ ​ User ID
 ​ ​ ​ ​ User Mention
 ​ ​ ​ ​ User Name#123
 """
-_GUILD_HELP = """
+_GUILD_HELP: Final[
+    str
+] = """
 Guild must be a valid version of one of the following:
 ​ ​ ​ ​ Guild ID
 ​ ​ ​ ​ Exact guild name
 """
```

Comment:
THIS IS PART 3/3 THAT NEED TO BE APPLIED TOGETHER WITH THIS.

```suggestion
_GUILD_HELP: Final[str] = _(
    """
Guild must be a valid version of one of the following:
​ ​ ​ ​ Guild ID
​ ​ ​ ​ Exact guild name
"""

_ = T_
```